### DISCUSSION TITLE
new(1.6.0) release date

### DISCUSSION THREAD

**Discussion by learingmmex** [moneymanagerex/moneymanagerex#5164] [Q&A] [answered]
i would like to ask when the new release date because the bugs in the current stable release is. 

another question: is it possible to prioritise "asset management bugs" resolution

thank you developers

**Comment by tactilis** [comment, marked as answer]
> i would like to ask when the new release date

The most accurate answer is: "When it is ready".

FYI, this has been the recent release cadence:
 
![image](https://user-images.githubusercontent.com/4060092/194513798-44f9904d-d589-40f9-ad63-bd7304687d62.png)

You could always re-install 1.5.20 if the bugs that are causing you problems were introduced in 1.5.21.

 
> is it possible to prioritise "asset management bugs" resolution

Too vague. Please provide the issue number(s) in which the bug(s) are described.

**Reply by MartinGoike** [reply]
Maybe it can be assessed a little more precisely based on the current status? It's just a question because 1.6.0 is eagerly awaited because of all the bug fixes.

**Reply by whalley** [reply]
v1.6.0 will hopefully be released today or tomorrow

**Comment by learingmmex** [comment]
thank you for your prompt reply, the bugs are:

management of assets #4412
Monthly Budget showing wrong "Actual" totals #5137

**Reply by whalley** [reply]
https://github.com/moneymanagerex/moneymanagerex/issues/5137 is already fixed ready to go in v1.6.0
https://github.com/moneymanagerex/moneymanagerex/issues/4412 is targeted at v1.6.0 but not sure if it will make it.
